llvm/lib/Transforms/Scalar/LoopInterchange.cpp

```diff
@@ -110,6 +111,7 @@ static bool populateDependencyMatrix(CharMatrix &DepMatrix, unsigned Level,
                     << " Loads and Stores to analyze\n");
 
   ValueVector::iterator I, IE, J, JE;
+  std::unordered_set<std::string> UniqueDepEntries;
```

Comment:
Using `unordered_set` is frowned upon as per [manual ](https://llvm.org/docs/ProgrammersManual.html#other-set-like-container-options)